## Goal
Target[] targets
---
## Result
uint32 target_id
float32 synthesis_time
float32 completion_time
---
## Feedback 
geometry_msgs/Pose2D curr_pose